docs/components/content/examples/ProgressExampleSlotIndicator.vue

```diff
@@ -0,0 +1,32 @@
+<script setup>
+const temp = ref(35)
+
+const color = computed(() => {
```

Comment:
Isn't a switch statement a better thing here with a default returning value

Reply:
I'm against `switch(true)` because, in this case, and `if` appear more clear.